thrift/src/main/java/com/linecorp/armeria/server/thrift/ThriftCallService.java

```diff
@@ -79,13 +80,13 @@ public static ThriftCallService of(Object implementation) {
      *                        {@code *.Iface} or {@code *.AsyncIface} service interface generated by
      *                        the Apache Thrift compiler
      */
-    public static ThriftCallService of(Map<String, ?> implementations) {
+    public static ThriftCallService of(Map<String, List<Object>> implementations) {
```

Comment:
- Could you change `List<Object>` to `Iterable<?>`?
- Could you retain the old method for backward compatibility? e.g.
  ```java
  implementations.entrySet().stream()
                 .map(e -> Maps.immutableEntry(e.getKey(), ImmutableList.of(e.getValue())))
                 .collect(toImmutableMap());
  ```